.github/CODEOWNERS

```diff
@@ -329,16 +329,16 @@ static/app/components/events/eventStatisticalDetector/                    @getse
 /static/app/views/organizationIntegrations                           @getsentry/product-owners-settings-integrations
 /static/app/views/settings/project/projectOwnership/                 @getsentry/issues
 
-/src/sentry/digests/                                                 @getsentry/issues
-/src/sentry/identity/                                                @getsentry/issues
+/src/sentry/digests/                                                 @getsentry/alerts-notifications
+/src/sentry/identity/                                                @getsentry/ecosystem
```

Comment:
is ecosystem being used again?

Reply:
Ecosystem is back but we don't own identity. I don't know what's the best way to handle this so tagging @leedongwei 

Maybe keep it with old enterprise so it gets routed to the rotation?

Reply:
Good call, that also keeps it consistent with the `/src/sentry/models/identity.py` line below which is also currently routed to enterprise